core/modules/layout/layout.theme.inc

```diff
@@ -250,9 +250,16 @@ function template_preprocess_block(&$variables) {
   $style = $block->style;
 
   $variables['classes'] = array();
+  $variables['attributes'] = array();
   $variables['title_prefix'] = '';
   $variables['title_suffix'] = '';
 
+  // Add the hero background image inline.
+  if (get_class($block) == 'BlockHero' && isset($block->settings['image_path'])) {
+    $variables['attributes']['style'] = 'background-image:url(' . $block->settings['image_path'] . ');';
+    $variables['classes'][] = 'hero-block';
+  }
```

Comment:
This could be moved to `template_preprocess_block__layout_hero_block()` but honestly I don't think that would be any cleaner than just including it here.

Why is `$block->settings['image_path']` required to add the `hero-block` class? Seems like it should always be added.